nevergrad/optimization/optimizerlib.py

```diff
@@ -401,7 +401,7 @@ def es(self) -> tp.Any:  # typing not possible since cmaes not imported :(
                     x0=self._rng.normal(size=self.dimension)
                     if self._random_init
                     else np.zeros(self.dimension, dtype=np.float),
-                    sigma0=self._scale,
+                    sigma0=0.3 * self._scale,
```

Comment:
shouldnt that be 0.3 as a default?
It feels weird to keep a factor between our parameters and CMA's

Reply:
Also, that's a big/breaking change that may lead to different optimizations for users (not sure if better or worse).
This may be especially a problem for those who possibly could have tuned the scale parameter, even though that is unlikely to have happened yet